.github/workflows/test.yml

```diff
@@ -73,3 +73,19 @@ jobs:
           run: |
             OSBUILD_TEST_STORE="${{ env.OSBUILD_TEST_STORE }}" \
             tox -e "py36" -- ${{ env.TEST_WORKERS }} test.run.test_assemblers
+
+  # This smoke test runs the unit tests directly on the runner and as a
+  # normal user - it is fast (2min) and should detect obvious issues
+  # (like from pr#1942)
+  unittests_as_user_smoke:
+    name: "Smoke run: unittest as normal user on default runner"
+    runs-on: ubuntu-24.04
```

Comment:
Do you think that it would be possible to run this job also inside the `osbuild-ci` container but switch to a non-privileged user before running the test? Alternatively, can we do it directly in the `test_suite` job?

Reply:
Maybe I should have made this more clear, but I think there is value in running the tests in a non-fedora environment, we had a recent issue that one of the tests assumed /var/empty to be available which is not the case on Debian based distros. The cost seems very low (it's just a smoke test) so I went this route (user+non-fedora distro in one test) but of course it could be done differently.

Reply:
Your comment about `/var/empty` makes sense. My reasoning was mainly that the `osbuild-ci` container already contains all the required dependencies for running tests. But I'm OK with this as a smoke test.